common/src/ethereum.rs

```diff
@@ -80,6 +80,7 @@ impl SignerOptions {
                 }
                 Long("walletconnect") => {
                     options.walletconnect = true;
+                    std::env::set_var("RAD_SIGNER_LEGACY", "true");
```

Comment:
Not sure this is the best way to pass a variable down :joy:

Reply:
Correct, this is temporary, and we'll eventually omit this once we move to v2.0. Also would have to add a lot of code, every place which calls `ethereum::transaction` has to find out if wallet is `WalletConnect`. there are 6 places currently which means at least 18 more lines, and an unnecessary flag on `ethereum::transaction` signature. 

Made a compromise here.